vividus-tests/src/main/resources/properties/configuration.properties

```diff
@@ -1,3 +1,5 @@
 configuration.profiles=web/headless/chrome
 configuration.suite=integration
 configuration.environments=
+
+bdd.variables.global.vividus-test-site-url=https://vividus-test-site.herokuapp.com
```

Comment:
`configuration.properties` is not the best place to configure global variables

Reply:
`vividus-tests/src/main/resources/properties/environment/envinroment.properties` ?